packages/core/src/pages/api/preview.ts

```diff
@@ -50,7 +50,10 @@ const handler: NextApiHandler = async (req, res) => {
     }
 
     // Enable Preview Mode by setting the cookies
-    res.setPreviewData(locator, { maxAge: 3600 })
+    res.setPreviewData(locator, {
+      maxAge: 3600,
+      path: (page.settings as Settings)?.seo?.slug,
```

Comment:
It's only in those places that the preview makes sense, right?

Reply:
maybe PDPs and PLPs as well?

Reply:
pdp should not be preview, because it is the final version, so to speak. So Home and PDP should be the pages where the images are previewed.

Reply:
It should affect those pages as well. If there's none it will be `undefined` which is the current behavior. 

I think that when you allow multiple templates per content-type they will be affected as well?

Reply:
Multiple templates didn't get the values from the `seo?.slug` value but from the `settings?.template?.value`, and this value is not always the slug (can be a category three or empty). So would it be better to add the `seo?.slug` for all pages as I suggest in [this task](https://vtex-dev.atlassian.net/browse/SFS-635).